### DISCUSSION TITLE
egl.h not generated by the web generator

### DISCUSSION THREAD

**Discussion by vtorri** [Dav1dde/glad#477] [General]
I want to have the C files for egl 1.5, so I set:
 * Specification : EGL
 * API egl: Version 1.5
 * Generate a loader is only checked

I click on `generate`, and when I open the archive, in `include` dir, the is only `include/glad/glad_egl.h` `include/EGL/eglpatform.h` (and khr file too)

is it normal ?

i tried to do that with the source flies, but there is no doc at all

**Comment by Dav1dde** [comment]
Yes that's normal. `glad_egl.h` replaces `egl.h`. All the loading code is `glad_egl.c`.

**Comment by vtorri** [comment]
then some tests should be fixed:

https://github.com/Dav1dde/glad/blob/glad2/test/c/compile/egl/default/001/test.c

1. `glad/egl.h` is included, not `glad/glad_egl.h`
2. `gladLoaderLoadEGL` and `gladLoaderLoadEGL` are not declared 

or am I missing something ?

same for the exemples: eg. https://github.com/Dav1dde/glad/blob/glad2/example/c/egl_x11/egl_x11.c : `egl.h` is included

**Reply by Dav1dde** [reply]
You're looking at `glad2` but you're using `glad1` (`master` branch).
